applications/ConvectionDiffusionApplication/python_scripts/python_solvers_wrapper_convection_diffusion.py

```diff
@@ -46,6 +46,8 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
         # Coupled mechanical-thermal solver
         elif (solver_type == "thermo_mechanically_coupled" or solver_type == "ThermoMechanicallyCoupled"):
             solver_module_name = "coupled_structural_thermal_solver"
+        elif (solver_type == "thermo_mechanically_coupled_substepping" or solver_type == "ThermoMechanicallyCoupledSubstepping"):
```

Comment:
```suggestion
        elif solver_type == "thermo_mechanically_coupled_substepping":
```

new solvers should only use the json style